### PR TITLE
Reference 'k6 cloud' instead of 'Load Impact' in docs and errors

### PR DESCRIPTION
**Pull request by mstoykov**
triggered by https://github.com/grafana/k6/pull/2916/files#r1117146117

### CODE REVIEW

File: output/cloud/output.go [hunk 1]
```diff
@@ -25,10 +25,10 @@ import (
 	"go.k6.io/k6/metrics"
 )
 
-// TestName is the default Load Impact Cloud test name
+// TestName is the default k6 Cloud test name
 const TestName = "k6 test"
 
-// Output sends result data to the Load Impact cloud service.
+// Output sends result data to the k6 cloud service.
```
Comment (imiric) [inline output/cloud/output.go:31]:
```suggestion
// Output sends result data to the k6 Cloud service.
```

File: cloudapi/errors.go [hunk 2]
```diff
@@ -8,9 +8,9 @@ import (
 )
 
 var (
-	ErrNotAuthorized    = errors.New("Not allowed to upload result to Load Impact cloud")
-	ErrNotAuthenticated = errors.New("Failed to authenticate with Load Impact cloud")
-	ErrUnknown          = errors.New("An error occurred talking to Load Impact cloud")
+	errNotAuthorized    = errors.New("not allowed to upload result to k6 cloud")
+	errNotAuthenticated = errors.New("failed to authenticate with k6 cloud")
+	errUnknown          = errors.New("an error occurred talking to k6 cloud")
```
Comment (imiric) [inline cloudapi/errors.go:13]:
```suggestion
	errNotAuthorized    = errors.New("not allowed to upload result to k6 Cloud")
	errNotAuthenticated = errors.New("failed to authenticate with k6 Cloud")
	errUnknown          = errors.New("an error occurred communicating with k6 Cloud")
```

File: cloudapi/config.go [hunk 3]
```diff
@@ -10,7 +10,7 @@ import (
 	"go.k6.io/k6/lib/types"
 )
 
-// Config holds all the necessary data and options for sending metrics to the Load Impact cloud.
+// Config holds all the necessary data and options for sending metrics to the k6 cloud.
```
Comment (imiric) [inline cloudapi/config.go:13]:
```suggestion
// Config holds all the necessary data and options for sending metrics to the k6 Cloud.
```

File: cloudapi/client.go [hunk 4]
```diff
@@ -23,7 +23,7 @@ const (
 	k6IdempotencyKeyHeader = "k6-Idempotency-Key"
 )
 
-// Client handles communication with Load Impact cloud API.
+// Client handles communication with k6 cloud API.
```
Comment (imiric) [inline cloudapi/client.go:26]:
```suggestion
// Client handles communication with the k6 Cloud API.
```

**Review by imiric — DISMISSED** [review]
Thanks!

LGTM, just minor suggestions.

**Review by imiric — APPROVED** [review]

**Comment by na--** [comment]
Something that would fit well in this PR, given that k6 can do a lot more than load testing nowadays, would be to remove the word "load" from here: https://github.com/grafana/k6/blob/1993ac1b6d47d3504631648d3cf02aebcd5adfbf/cmd/run.go#L376-L377

**Review by na-- — APPROVED** [review]
thanks! :bow:

**Review by imiric — APPROVED** [review]

**Pull request merged**